### ISSUE TITLE
Does not work with package:individually:true

### ISSUE THREAD

**Issue by SiarheiMelnik** [elastic-coders/serverless-webpack#32]
1. I see Error Serverless: Uploading function .zip files to S3...
   
   Error --------------------------------------------------
      ENOENT: no such file or directory .webpack.serverless\servicename-funtionName.zip'
2. In zip archive I see all entry points but want only required by function handler in serverless.yml

**Comment by Si1kIfY** [comment]
I can reproduce this MacOSX - v1.0.0-beta.2.2

```
Error --------------------------------------------------
     ENOENT: no such file or directory, open '/Users/csilk/projectName/.webpack/.serverless/functionName.zip'
```

Seems to be building them properly into the `.serverless` dir just can't reference them which is strange

Could it be to do with setting the `serverless.config.servicePath` step in `compile.js`?

Am happy to throw my time at fixing this as it's a blocker for me just don't really know where to start

Cheers

**Comment by Si1kIfY** [comment]
Looks like this is fixed with #48 

Waiting for merge

**Comment by thenikso** [comment]
merged. keep an eye for v1.0.0 which should be released soon
